docs/api_reference.md

```diff
@@ -217,6 +218,14 @@ These are all available on the global `LookerCharts.Utils` object.
 
 	This function accepts a cell and returns a Looker advanced filter syntax string that would match the value of this cell.
 
+- `LookerCharts.Utils.crossfilter({row, pivot, event})`
+	
+	This function accepts a row, pivot, or event and is used to toggle crossfilters in a visualization.
+
+- `LookerCharts.Utils.isCrossfilterSelected(row, pivot?)`
```

Comment:
This was also changed super recently and is now called `getCrossfilterSelection` so `LookerCharts.Utils.getCrossfilterSelection(row, pivot?)`

This change was made to reflect the fact that this actually returns an enum (which we should also note here). Specifically: `CrossfilterSelection {NONE, SELECTED, UNSELECTED}`. Ultimately users need to check for 1 rather than a truthy condition. A result of 0 means "NONE" and 2 is "UNSELECTED". Here's a small example:

```
... conditionally apply fill color if being crossfiltered ...

d3.select("#myElement")
  .attr("fill", function (d) {
    if (details.crossfilterEnabled) {
      if (LookerCharts.Utils.getCrossfilterSelection(d.row) === 1) {
        return d.color
      } else {
        return "#DEE1E5" //Hex used for unselected elements in native visualizations
      }
    } else {

...apply normal colors if crossfiltering not enabled ...
```